gulpfile.js

```diff
@@ -1,26 +1,46 @@
 var elixir = require('laravel-elixir');
```

Comment:
@vinicius73  I have a question for you, webpack/laravel-mix its works in Laravel 5.1? So, this branch is refer to Laravel 5.1